LandSandBoat/server
# Add in missing items_equipment and items_weapons.

Also start adding in changes from vitus0123456



**_I affirm:_**

## What does this pull request do?

Adds item_basic, item_equipment, item_weapon missing items, as well as many missing item mods,  latents, and pet mods.
It helps add the beginning of the mods that [vitus0123456](https://github.com/vitus0123456) in https://github.com/LandSandBoat/server/discussions/3839 worked on.

## Steps to test these changes

!additem for changed items. Make sure they can add, equip, shows relevant information, and adds relevant mods.

PS: Sorry for the length of this PR. I messed up on my last PR so am submitting them all at once. Not many items were changed, but there are many lines of change in the item_mods.sql file since the items I am working on have so many mods per item.
